docs/source/endpoints/translations.md

```diff
@@ -125,7 +125,11 @@ This endpoint returns the proper placement for the newly created translation:
 
 ## Expansion
 
-This endpoint can be used with the {doc}`expansion` mechanism which allows getting additional information about a content item in one query, avoiding unnecessary requests.
+This service can be used with the {doc}`expansion` mechanism which allows getting additional information about a content item in one query, avoiding additional requests.
+
+As soon as plone.app.multilingual is installed, a content item is translatable.
+Which means it provides the ITranslatable interface.
```

Comment:
A content type is translatable if it has the `plone.translatable` behavior enabled. plone.app.multilingual enables this behavior for all content types at the time that it is installed. But if other content types are added later then it would be necessary to enable the behavior for them also.